pipelines/sea-surface-mapping-r/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
```

Comment:
since we don't have an `-e` on this file, we won't get a failure if any of the commands fail